freqtrade/exchange/exchange.py

```diff
@@ -959,6 +1014,7 @@ def cancel_order(self, order_id: str, pair: str) -> Dict:
         try:
             order = self._api.cancel_order(order_id, pair)
             self._log_exchange_response('cancel_order', order)
+            order = self._order_contracts_to_amount(order)
```

Comment:
You will need to be careful with `cancel_order` calls.

I've rolled back the changes to `test_cancel_order` - that test MUST pass without modification.

Order cancellation is not guaranteed to contain anything - so we could get a response like `{'result': 'success'}` (not containing any symbol or amount) - and that response cannot have freqtrade crash.